src/test/java/ru/semrush/appmanager/BlogHelper.java

```diff
@@ -0,0 +1,68 @@
+package ru.semrush.appmanager;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.asserts.SoftAssert;
+import ru.semrush.model.PostData;
+import ru.semrush.model.Posts;
+
+import java.util.List;
+
+public class BlogHelper extends HelperBase {
+
+    public BlogHelper(ApplicationManager app) {
+        super(app);
+    }
+
+
+    public Posts all(PostType postType) {
+        Posts posts = new Posts();
+        if (postType == PostType.LARGE_CARDS) {
+            List<WebElement> elements = wd.findElements(By.xpath("//div[@data-test='post-card-large']"));
```

Comment:
`xpath` does not allow you using full benefits of data-test. Firstly data-test needs cause of any element moving and changing do not broke tests. In your case it does not work and tests will be broken.